kivy/base.py

```diff
@@ -229,11 +230,32 @@ def post_dispatch_input(self, etype, me):
             if me in self.me_list:
                 self.me_list.remove(me)
 
+        if hasattr(me, 'name') and me.name == 'hover':
```

Comment:
Can you make a single variable for hasattr(me, 'name') and me.name == 'hover' ? As it is used in 2 if.

Reply:
I can use `if getattr(me, 'name', None) == 'hover':` instead of that.